bot/cogs/monstersurvey.py

```diff
@@ -0,0 +1,161 @@
+            default_embed.add_field(
+                name='.monster vote monster_name',
+                value='Vote for a specific monster. You can vote more than once, but you can only vote for one monster'
+                      'at a time.',
+                inline=False
+            )
+            default_embed.add_field(
+                name='.monster leaderboard',
+                value='Which monster has the most votes? This command will tell you.',
+                inline=False
+            )
+            return await ctx.send(embed=default_embed)
+
+    @monster_group.command(
+        name='vote'
+    )
+    async def monster_vote(self, ctx: Context, name: Optional[str] = None):
+        """Casts a vote for a particular monster, or displays a list of monsters that can be voted for
+        if one is not given."""
+        vote_embed = Embed(
+            name='Monster Voting',
+            color=0xFF6800
+        )
+        if name is None or name.lower() not in self.voter_registry.keys():
+            if name is not None:
+                vote_embed.description = f'You cannot vote for {name} because it\'s not in the running.'
+            vote_embed.add_field(
+                name='Use `.monster show {monster_name}` for more information on a specific monster',
+                value='or use `.monster vote {monster}` to cast your vote for said monster.',
+                inline=False
+            )
+            vote_embed.add_field(
+                name='You may vote for or show the following monsters:',
+                value=f"{', '.join(self.voter_registry.keys())}"
+            )
+            return await ctx.send(embed=vote_embed)
+        for monster in self.voter_registry.keys():
+            if ctx.author.id in self.voter_registry[monster]['votes']:
+                if name != monster:
+                    self.voter_registry[monster]['votes'].remove(ctx.author.id)
+                    break
+                else:
+                    vote_embed.add_field(
+                        name='Vote unsuccessful.',
+                        value='You already voted for this monster. '
+                              'If you want to change your vote, use another monster.',
+                        inline=False
+                    )
+                    await ctx.send(embed=vote_embed)
+                    await asleep(.5)
+                    return await ctx.invoke(self.monster_vote)
+        self.voter_registry[name]['votes'].append(ctx.author.id)
+        vote_embed.add_field(
+            name='Vote successful!',
+            value=f'You have successfully voted for {self.voter_registry[name]["full_name"]}!',
+            inline=False
+        )
+        vote_embed.set_thumbnail(url=self.voter_registry[name]['image'])
+        self.json_write()
+        return await ctx.send(embed=vote_embed)
+
+    @monster_group.command(
+        name='show'
+    )
+    async def monster_show(self, ctx: Context, name: str):
```

Comment:
Need a default argument for `name`, otherwise an exception will be thrown if no monster name is provided.